src/ml/neural_net/tf_compute_context.hpp

```diff
@@ -45,7 +45,11 @@ EXPORT class tf_compute_context: public compute_context {
       const float_array_map& config, const float_array_map& weights) override;
 
   std::unique_ptr<model_backend> create_drawing_classifier(
+<<<<<<< HEAD
```

Comment:
This looks like a bad git merge.

Reply:
Thanks for it pointing out, not sure how I missed this. Fixed now.